packs/journals/gm-screen.json

```diff
@@ -1084,7 +1084,7 @@
d>@Localize[PF2E.TraitDescriptionUnarmed]</td></tr><tr><td><section class=\"traits\"><p>Versatile</p></section></td><td>@Localize[PF2E.TraitDescriptionVersatile]</td></tr><tr><td><section class=\"traits\"><p>Volley</p></section></td><td>@Localize[PF2E.TraitDescriptionVolley]</td></tr></tbody></table><p><span style=\"float:right\"><em>Pathfinder Player Core pg. 282-283</em></span></p><p></p><h3>Summon Trait</h3><p>Spells with the summon trait can conjure creatures, typically ones with a particular trait. Such creatures can be found in Monster Core and similar books. Unless noted otherwise, the creature must be common, it gains the summoned trait, and it must appear in an unoccupied space in range large enough to contain it. The highest level of creature the spell can summon depends on the rank of the spell, as listed below. The spell can still summon a creature of a lower level if you so choose. These rules apply only to spells that have the summon trait; other spells that call or conjure items or beings but that don't have the trait, like summon instrument, work as explained in the spell.</p><table border=\"1\" class=\"pf2e remaster\"><thead><tr style=\"text-align:right\"><th>Spell Rank</th><th>Maximum Creature Level</th></tr></thead><tbody><tr><td>1st</td><td>–1</td></tr><tr><td>2nd</td><td>1</td></tr><tr><td>3rd</td><td>2</td></tr><tr><td>4th</td><td>3</td></tr><tr><td>5th</td><td>5</td></tr><tr><td>6th</td><td>7</td></tr><tr><td>7th</td><td>9</td></tr><tr><td>8th</td><td>11</td></tr><tr><td>9th</td><td>13</td></tr><tr><td>10th</td><td>15</td></tr></tbody></table><p><span style=\"float:right\"><em>Pathfinder Player Core pg. 360</em></span></p><p></p><h3>General Traits</h3><table border=\"1\" class=\"pf2e remaster\"><thead><tr style=\"text-align:right\"><th>Trait</th><th>Description</th></tr></thead><tbody><tr><td><section class=\"traits\"><p>Additive</p></section></td><td>@Localize[PF2E.TraitDescriptionAdditive]</td></tr><tr><td><section class=\"traits\"><p>Deadly</p></section></td><td>@Localize[PF2E.TraitDescriptionDeadly]</td></tr><tr><td><section class=\"traits\"><p>Death</p></section></td><td>@Localize[PF2E.TraitDescriptionDeath]</td></tr><tr><td><section class=\"traits\"><p>Flourish</p></section></td><td>@Localize[PF2E.TraitDescriptionFlourish]</td></tr><tr><td><section class=\"traits\"><p>Finesse</p></section></td><td>@Localize[PF2E.TraitDescriptionFinesse]</td></tr><tr><td><section class=\"traits\"><p>Move</p></section></td><td>@Localize[PF2E.TraitDescriptionMove]</td></tr><tr><td><section class=\"traits\"><p>Press</p></section></td><td>@Localize[PF2E.TraitDescriptionPress]</td></tr><tr><td><section class=\"traits\"><p>Spellshape</p></section></td><td>@Localize[PF2E.TraitDescriptionSpellshape]</td></tr><tr><td><section class=\"traits\"><p>Stance</p></section></td><td>@Localize[PF2E.TraitDescriptionStance]</td></tr></tbody></table><p><em>Section: Playing the Game</em><span style=\"float:right\"><em>Pathfinder Player Core</em></span></p>",
```

Comment:
Give the summon trait it's own page and keep the old links as is. We can re-id the new page.